depends/packages/openssl.mk

```diff
@@ -0,0 +1,45 @@
+# Pretty much taken verbatim from Bitcoin Core
```

Comment:
I believe this has been updated recently to 1.0.1m, which has several critical security fixes. Please update.